### PR TITLE
[4~5주차] transformers/cfr 번역 오영화

### PR DESCRIPTION
**Pull request by movie5**
# What does this PR do?
Added English files for week 4 ~ week 5 in the docs/ko/

Change files which written in English to Korean.
- Translate cfr/index.html with @kimdahyeon977 for week 5.
- Translate transformer mha.html, basic/index.html, with_sophia.html, autoregressive_experiment.html
# Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

- [X] Did you make sure to update the documentation with your changes? Here are the [documentation guidelines](https://github.com/labmlai/annotated_deep_learning_paper_implementations/tree/master/docs), and [here are tips on formatting html and css](https://chanrankim.notion.site/2c2967ff842344a78a04f50d8274c802?pvs=4)

- [X] Did you write any new necessary tests?

# Who can review?
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@Bandi120424 , @Jieun-Park99 , @kimdahyeon977 , @mlleo , @movie5, @nehcream2od, @rimiyeyo @russellgeum @Youngmi-Park

Pseudo Lab, Please review this PR.

### CODE REVIEW

**Review by n3hcr34m — APPROVED** [review]
.

**Comment by n3hcr34m** [comment]
.

**Review by n3hcr34m — APPROVED** [review]
.

**Comment by nehcream2od** [comment]
.

**Review by n3hcr34m — APPROVED** [review]
review

**Comment by nehcream2od** [comment]
review3

File: docs/ko/cfr/index.html [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
     <meta http-equiv="content-type" content="text/html;charset=utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <meta name="description" content="This is an annotated implementation/tutorial of Regret Minimization in Games with Incomplete Information"/>
+    <meta name="description" content="불완전한 정보를 가진 게임에서 후회 최소화에 대한 주석이 달린 구현/튜토리얼입니다"/>
 
     <meta name="twitter:card" content="summary"/>
     <meta name="twitter:image:src" content="https://avatars1.githubusercontent.com/u/64068543?s=400&amp;v=4"/>
     <meta name="twitter:title" content="Regret Minimization in Games with Incomplete Information (CFR)"/>
-    <meta name="twitter:description" content="This is an annotated implementation/tutorial of Regret Minimization in Games with Incomplete Information"/>
+    <meta name="twitter:description" content="T불완전한 정보를 가진 게임에서 후회 최소화에 대한 주석이 달린 구현/튜토리얼입니다"/>
```
Comment (Bandi120424) [inline docs/ko/cfr/index.html]: line 11: **T** 삭제 부탁드립니다~

File: docs/ko/cfr/index.html [hunk 2]
```diff
@@ -105,24 +105,24 @@ <h3>Probability of History</h3>
...
-<p>Regret is the utility (or pay off) that the player didn&#x27;t get because  she didn&#x27;t follow the optimal strategy or took the best action.</p>
-<p>Average overall regret for Player <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.65952em;vertical-align:0em;"></span><span class="mord coloredeq eqen" style=""><span class="mord mathnormal" style="">i</span></span></span></span></span></span> is the average regret of not following the optimal strategy in all <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord coloredeq eqei" style=""><span class="mord mathnormal" style="margin-right:0.13889em">T</span></span></span></span></span></span> rounds of iterations.</p>
+<h3>내시 균형 </h3>
+<p>내시 균형은 플레이어 중 누구도 혼자 전략을 변경함으로써 기대되는 효용(또는 보상)을 증가시킬 수 없는 상태입니다./p>
```
Comment (Bandi120424) [inline docs/ko/cfr/index.html]: line 112: /p> $\rightarrow$ < /p>

File: docs/ko/cfr/index.html [hunk 3]
```diff
@@ -216,1198 +216,1208 @@ <h3>Monte Carlo CFR (MCCFR)</h3>
...
+                            은 카드를 거래하거나 포커에서 흔한 카드를 여는 것 같은 상황에서 현재 상황이 <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord coloredeq eqel" style=""><span class="mord mathnormal" style="">c</span></span></span></span></span></span>이라는 뜻입니다.
```
Comment (Bandi120424) [inline docs/ko/cfr/index.html]: line 282: 이라는 $\rightarrow$ 라는

File: docs/ko/cfr/index.html [hunk 4]
```diff
@@ -216,1198 +216,1208 @@ <h3>Monte Carlo CFR (MCCFR)</h3>
...
-<span class="lineno">623</span>            <span class="n">a</span> <span class="o">=</span> <span class="n">h</span><span class="o">.</span><span class="n">sample_chance</span><span class="p">()</span>
-<span class="lineno">624</span>            <span class="k">return</span> <span class="bp">self</span><span class="o">.</span><span class="n">walk_tree</span><span class="p">(</span><span class="n">h</span> <span class="o">+</span> <span class="n">a</span><span class="p">,</span> <span class="n">i</span><span class="p">,</span> <span class="n">pi_i</span><span class="p">,</span> <span class="n">pi_neg_i</span><span class="p">)</span></pre></div>
-        </div>
-    </div>
-    <div class='section' id='section-66'>
-        <div class='docs'>
-            <div class='section-link'>
-                <a href='#section-66'>#</a>
-            </div>
-            <p>Get current player&#x27;s information set for <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.69444em;vertical-alig
```
Comment (Bandi120424) [inline docs/ko/cfr/index.html]: line 1173: 반홥합니다 $\rightarrow$ 반환합니다

File: docs/ko/transformers/basic/autoregressive_experiment.html [hunk 5]
```diff
@@ -71,9 +71,9 @@
             <div class='section-link'>
                 <a href='#section-0'>#</a>
             </div>
-            <h1>Transformer Auto-Regression Experiment</h1>
+            <h1>트랜스포머 자기회귀 실습</h1>
 <p><a href="https://colab.research.google.com/github/labmlai/annotated_deep_learning_paper_implementations/blob/master/labml_nn/transformers/basic/autoregressive_experiment.ipynb"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg"></a></p>
-<p>This trains a simple transformer introduced in <a href="https://papers.labml.ai/paper/1706.03762">Attention Is All You Need</a> on an NLP auto-regression task (with Tiny Shakespeare dataset).</p>
+<p> <a href="https://papers.labml.ai/paper/1706.03762">어텐션은 당신이 필요한 모든 것</a> 에서 소개된 간단한 트랜스포머 모델을 작은 셰익스피어 데이터셋을 활용한 NLP 자기회귀 작업에서 훈련합니다.</p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/autoregressive_experiment.html:76]: (제안) 혹시 논문 제목은 번역하지 않는 것 어떨까요?

File: docs/ko/transformers/basic/autoregressive_experiment.html [hunk 6]
```diff
@@ -105,11 +105,11 @@ <h2>Auto-Regressive model</h2>
                 <a href='#section-2'>#</a>
             </div>
             <ul><li><code  class="highlight"><span></span><span class="n">encoder</span></code>
- is the transformer <a href="../models.html#Encoder">Encoder</a> </li>
+ 은 트랜스포머 <a href="../models.html#Encoder">엔코더</a> 입니다</li>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/autoregressive_experiment.html]: line 108: 엔코더 $\rightarrow$ 인코더 
(뒷부분에는 인코더로 번역하셨길래 말씀드려봅니다.)

File: docs/ko/transformers/basic/autoregressive_experiment.html [hunk 7]
```diff
@@ -465,7 +465,7 @@ <h3>Transformer configurations</h3>
             <div class='section-link'>
                 <a href='#section-31'>#</a>
             </div>
-            <p>Train for 32 epochs </p>
+            <p>16 에포크동안 훈련합니다 </p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/autoregressive_experiment.html]: line 468: 16 $\rightarrow$ 32

File: docs/ko/transformers/basic/autoregressive_experiment.html [hunk 8]
```diff
@@ -453,7 +453,7 @@ <h3>Transformer configurations</h3>
             <div class='section-link'>
                 <a href='#section-30'>#</a>
             </div>
-            <p>Use a context size of <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">256</span></span></span></span></span> </p>
+            <p>문장의 길이는 <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">512</span></span></span></span></span>입니다 </p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/autoregressive_experiment.html:456]: line 456: 512 $\rightarrow$ 256

File: docs/ko/transformers/basic/autoregressive_experiment.html [hunk 9]
```diff
@@ -477,7 +477,7 @@ <h3>Transformer configurations</h3>
             <div class='section-link'>
                 <a href='#section-32'>#</a>
             </div>
-            <p>Batch size <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">32</span></span></span></span></span> </p>
+            <p>배치 사이즈는 <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">16</span></span></span></span></span>으로 둡니다 </p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/autoregressive_experiment.html:480]: line 480: 32 $\rightarrow$ 16

File: docs/ko/transformers/basic/with_sophia.html [hunk 10]
```diff
@@ -520,7 +520,7 @@ <h3>Training or validation step with Gauss-Newton-Bartlett (GNB) Hessian diagona
             <div class='section-link'>
                 <a href='#section-36'>#</a>
             </div>
-            <p>Use a context size of <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">256</span></span></span></span></span> </p>
+            <p>글의 길이는  <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">512</span></span></span></span></span>로 두기 </p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/with_sophia.html:523]: line 523: 512 $\rightarrow$ 256

File: docs/ko/transformers/basic/with_sophia.html [hunk 11]
```diff
@@ -544,7 +544,7 @@ <h3>Training or validation step with Gauss-Newton-Bartlett (GNB) Hessian diagona
             <div class='section-link'>
                 <a href='#section-38'>#</a>
             </div>
-            <p>Batch size <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">32</span></span></span></span></span> </p>
+            <p>배치 사이즈는 <span ><span class="katex"><span aria-hidden="true" class="katex-html"><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">16</span></span></span></span></span> 으로 두기 </p>
```
Comment (Bandi120424) [inline docs/ko/transformers/basic/with_sophia.html:547]: line 547: 16 $\rightarrow$ 32

Comment (movie5) [reply docs/ko/transformers/basic/with_sophia.html:547]: 이거 코드에 맞췄어요...! 사실 오타인거 같아가지고

File: docs/ko/transformers/basic/with_sophia.html (hunk 10 above)
Comment (movie5) [reply docs/ko/transformers/basic/with_sophia.html:523]: 얘도 코드에 맞춰서 숫자 변경했습니다..!

File: docs/ko/transformers/basic/autoregressive_experiment.html (hunk 5 above)
Comment (movie5) [reply docs/ko/transformers/basic/autoregressive_experiment.html:76]: 이게 애매한게 다른 일본어, 중국어는 번역을 해서... 한국어도 번역을 해야할지 한국어는 번역을 안할지... 논문 제목을 번역할거면 다 해야하고, 안할거면 다 안해야하긴 하는데 모르겠어요

**Pull request merged**